### PR TITLE
Make last command available in clickhouse-client interactive mode

### LINKED ISSUE #40280: / (shortcut) to repeat the last (previous) SQL command  (clickHouse-client)
Oracle cli client (sqlplus) has a neat feature, if you put `/[enter]` in the prompt, it re-executes the last SQL.

```sql
:) select 1;
┌─1─┐
│ 1 │
└───┘

:) /
┌─1─┐
│ 1 │
└───┘
```

currently `<up><enter>` pollutes the screen by the SQL text and can contain SQL script, not a single query. Probably echo should be disabled in this case as well.

### PR DESCRIPTION
**Pull request by rschu1ze**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
The interactive mode in clickhouse-client now interprets `.` and `/` as "run the last command"

Fixes #40280

### CODE REVIEW

File: src/Client/ClientBase.cpp [hunk 1]
```diff
@@ -2034,10 +2039,21 @@ void ClientBase::runInteractive()
             }
         }
 
+        for (const auto & alias : repeat_last_input_aliases)
+        {
+            if (input == alias)
+            {
+                input  = last_input;
+                break;
+
```
Comment (evillique) [inline src/Client/ClientBase.cpp:2048]:
```suggestion
```

File: src/Client/ClientBase.cpp [hunk 2]
```diff
@@ -2016,7 +2020,8 @@ void ClientBase::runInteractive()
             has_vertical_output_suffix = true;
         }
 
-        for (const auto& [alias, command] : backslash_aliases)
+
```
Comment (evillique) [inline src/Client/ClientBase.cpp:2023]:
```suggestion
```

**Review by evillique — APPROVED** [review]
Looks great, just some stylistic preferences :)

**Comment by rschu1ze** [comment]
Thanks! Done.

**Comment by azat** [comment]
>The interactive mode in clickhouse-client now interprets . and / as "run the last command"

BTW, `bash` uses `!!` or `!-N` where `N` is repeat last's `N`th command, what do you think about this cases too?

**Comment by rschu1ze** [comment]
Alexey [didn't like](https://github.com/ClickHouse/ClickHouse/issues/40280#issuecomment-1221585885) `!!` very much. 

> [...] `!-N` where `N` is repeat last's `N`th command [...]

For this to work, user must keep a mental history of the N last commands. At least I rather like to scroll visually through the history. And that functionality is already there using `<Up>`/`<Down>` and `<Ctrl>+p`/`<Ctrl>+n`.

I'd say let's keep things as is for now. And if someone has a compelling need for `!-N`, this can be implemented quite easily on top of this PR.

**Comment by alexey-milovidov** [comment]
@rschu1ze Let's add a test with `expect`. Similar tests exist.

**Comment by rschu1ze** [comment]
@alexey-milovidov Yepp, will do. (You surely mean `tests/queries/0_stateless/*.expect` tests. At least these are the only tests I could find for clickhouse-client.)

**Comment by alexey-milovidov** [comment]
Yes, and also somethines we embed expect script into `.sh`.

**Pull request merged**